src/test/java/org/apache/commons/text/RandomStringGeneratorTest.java

```diff
@@ -212,6 +212,14 @@ public void testSelectFromCharVarargs() {
         }
     }
 
+    @Test
+    public void testSelectFromNullCharVarargs() {
+        final String str = "abc";
+        final RandomStringGenerator generator = new RandomStringGenerator.Builder().selectFrom(null).build();
+        final String randomText = generator.generate(5);
+        assertThat(randomText.isEmpty()).isFalse();
```

Comment:
You need to assert the results are correct, not that any old output was generated.